lib/mixins/link-getters.js

```diff
@@ -40,7 +40,10 @@ export default function mixinLinkGetters (items, includes) {
     const fieldKeys = Object.keys(fields)
     fieldKeys.forEach((fieldKey) => {
       const field = fields[fieldKey]
-      if (Array.isArray(field)) {
+      const unresolved = errors.find((error) => error.details.id === field.sys.id)
```

Comment:
Is it possible to have different errors besides resolving errors?

If yes, how do we wanna handle them? Just remove all fields with errors? (Sounds good to me, just want to clarify)